source/documentation/guides/security_process.html.md.erb

```diff
@@ -1,45 +1,96 @@
 ---
 title: Security in our Process
-last_reviewed_on: 2024-08-19
+last_reviewed_on: 2025-07-08
 review_in: 13 weeks
 ---
 
 # <%= current_page.data.title %>
 
-Our aim is to make our products as secure as possible while maintaining usability for end users. This is in line with the [GDS service standard](https://www.gov.uk/service-manual/service-standard/point-9-create-a-secure-service). You should be aware of the [MOJ Security Guidance](https://ministryofjustice.github.io/security-guidance/#cyber-and-technical-security-guidance), which will help with specific policy issues at an MOJ level and follow the [National Cyber Security Centre](https://www.ncsc.gov.uk/) for updates at a government level. 
+## Secure By Design
+
+### Policy
+
+Our aim within OPG is to make our products as secure as possible while maintaining usability for end users.
+
+All central government departments and arm's length bodies (ALBs) must incorporate effective security practices and be moving towards meeting the [Secure by Design policy](https://www.security.gov.uk/policy-and-guidance/secure-by-design/policy/) when delivering and building digital services and technical infrastructure.
```

Comment:
Might be worth adding a note that where products existed before this policy, we will attempt to bring them in line but there may be historical gaps (for example where it mandates security is part of an original business case)